### PR TITLE
added conditional download resume feature

### PR DESCRIPTION
**Pull request by joanjeremiah**
I have implemented 'Download Resume' feature . Just let me know if any changes required

### CODE REVIEW

File: src/components/avatar-card/index.jsx [hunk 1]
```diff
@@ -55,6 +55,9 @@ const AvatarCard = ({ profile, loading, avatarRing }) => {
               : profile.bio}
           </div>
         </div>
+        { resume?.fileName &&
+          <a href= { resume.fileName } target="_blank" className='btn mt-4' download>Download my resume</a>
```
Comment (arifszn) [inline src/components/avatar-card/index.jsx:59]:
```suggestion
          <a
            href={resume.fileName}
            target="_blank"
            className="btn btn-outline btn-primary text-xs mt-6"
            download
          >
            Download Resume
          </a>
```

**Review by arifszn — CHANGES_REQUESTED** [review]
Thanks for the PR. A small modification is required. Also, please update the readme.

**Review by joanjeremiah — COMMENTED** [review]
I made the suggested changes in the 'Download Resume' button and also update the readme markdown.

**Comment by arifszn** [comment]
Looks good. Can you please run the command `npm run lint:fix:`? Also, are you participating in `hacktoberfest`? Do I need to do anything after merging it so that it counts against you?

**Comment by joanjeremiah** [comment]
> Looks good. Can you please run the command `npm run lint:fix:`? Also, are you participating in `hacktoberfest`? Do I need to do anything after merging it so that it counts against you?

I have run the lint fix command. Do I need to make another pull request ? And yes I am participating in `hacktoberfest` . I believe simply merging my pull request is enough for it to count . I will let you know if it was not taken into the count.

**Comment by joanjeremiah** [comment]
I don't know why the `run prettier` command fails . Please let me know what I need to do.

**Comment by arifszn** [comment]
> I don't know why the `run prettier` command fails . Please let me know what I need to do.

Maybe try running `npm run prettier:fix`.

**Comment by joanjeremiah** [comment]
> > I don't know why the `run prettier` command fails . Please let me know what I need to do.
> 
> Maybe try running `npm run prettier:fix`.

I tried running the command and it works !

**Pull request merged**